packages/hooks/src/useEventListener.ts

```diff
@@ -38,12 +42,18 @@ export default function useEventListener(
       return;
     }
 
-    const callback = e => memoizedEventCallback.current(e);
+    const callback = (e: GlobalEventHandlersEventMap[Type]) =>
```

Comment:
Same comment as above – if we type this differently, can we omit typing where it's used on line 49?